app/models/alchemy/language.rb

```diff
@@ -26,20 +29,30 @@ class Language < ActiveRecord::Base
     validates_uniqueness_of :language_code, scope: [:site_id, :country_code]
     validate :presence_of_default_language
     validate :publicity_of_default_language
-    has_many :pages
-    belongs_to :site
-    after_destroy :delete_language_root_page
-    validates_format_of :language_code, with: /\A[a-z]{2}\z/, if: -> { language_code.present? }
-    validates_format_of :country_code, with: /\A[a-z]{2}\z/, if: -> { country_code.present? }
+
+    validates_format_of :language_code, with: /\A[a-z]{2}\z/,
```

Comment:
Rails/Validation: Prefer the new style validations `validates :column, format: value` over `validates_format_of`.